packages/token-lists/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import buildList from "./buildList";
+
+console.log(JSON.stringify(buildList(), null, 2));
```

Comment:
rather use `.info` the user of `.log` is disabled in all our codebase (for debug only)